README.adoc

```diff
@@ -99,12 +99,12 @@ There are several modules in Spring Boot. Here is a quick overview:
 
 
 === spring-boot
-The main library providing features that support the other parts of Spring Boot. These include:
+The main library provides features that support the other parts of Spring Boot. These include:
```

Comment:
For consistency, I don't think should be changed.

Reply:
I changed here because it appears that this sentence is using an incorrect form of the verb "providing". But for consistency, I may change it as per your guidance.